docs_2.0/manual_EN/3.quick-start/4.nebula-graph-crud.md

```diff
@@ -501,12 +502,19 @@ CREATE {TAG | EDGE} INDEX [IF NOT EXISTS] <index_name>
 ON {<tag_name> | <edge_name>} (prop_name_list);
 ```
 
+Rebuild an index:
+
+```ngql
+REBUILD {TAG | EDGE} INDEX <index_name>
```

Comment:
no offline this time?

Reply:
no, confirmed with the dev. No offline in the future, either.